### PR TITLE
package updates

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
Pull Request
**Are you a current Fivetran customer?** 

Internal
**What change(s) does this PR introduce?** 

We are applying standardization updates to be more consistent in our documentation.
**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [x] Yes (please provide breaking change details below.)
- [] No  (please provide an explanation as to how the change is non-breaking below.)
This won't be necessarily be breaking, but this will be a major change to the look and feel of our package.
**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [ ] Yes, Issue/Feature [link bug/feature number here]
- [x] No 

**How did you test the PR changes?** 

 

- [x] CircleCi  
- [] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
📚 :

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,54 +31,55 @@ This package contains transformation models, designed to work simultaneously wit
...
-Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
+## Step 1: Prerequisites
+To use this dbt package, you must have the following:
 
-Include in your `packages.yml`
+- At least one Fivetran Shopify connector syncing data into your destination.
+- A **BigQuery**, **Snowflake**, **Redshift**, **Databricks**, or **PostgreSQL** destination.
 
+## Step 2: Install the package
+Include the following shopify_source package version in your `packages.yml` file:
```
Comment (fivetran-joemarkiewicz) [inline README.md:43]:
```suggestion
Include the following shopify package version in your `packages.yml` file:
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,26 @@
...
+
+This package models Shopify data from [Fivetran's connector](https://fivetran.com/docs/applications/shopify). It uses data in the format described by [this ERD](https://fivetran.com/docs/applications/shopify#schemainformation) and builds off the output of our [Shopify source package](https://github.com/fivetran/dbt_shopify_source).
 
 The main focus of the package is to transform the core object tables into analytics-ready models, including a cohort model to understand how your customers are behaving over time.
 
-## Models
-
-This package contains transformation models, designed to work simultaneously with our [Shopify source package](https://github.com/fivetran/dbt_shopify_source). A dependency on the source package is declared in this package's `packages.yml` file, so it will automatically download when you run `dbt deps`. The primary outputs of this package are described below.
+The following table provides a detailed list of all models materialized within this package by default. 
+> TIP: See more details about these models in the package's [dbt docs site](https://fivetran.github.io/dbt_shopify/#!/overview?g_v=1&g_e=seeds).
```
Comment (fivetran-joemarkiewicz) [inline README.md:23]: Ca you update the links in the table below to reference the dbt docs instead of the sql files.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+# dbt_shopify v0.7.0
+🎉 Applying Package Standardization 🎉
+We are applying standardization updates to be more consistent across our documentation.
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:3]:
```suggestion
## 🎉 Documentation and Feature Updates
- Updated README documentation updates for easier navigation and setup of the dbt package
- Included `shopify_[source_table_name]_identifier` variable within the Shopify source package for additional flexibility within the package when source tables are named differently.
```

File: README.md [hunk 4]
```diff
@@ -18,54 +31,55 @@ This package contains transformation models, designed to work simultaneously wit
...
+## Step 4: Disable models for non-existent sources
+This package was designed with the intention that users have all relevant Shopify tables being synced by Fivetran. However, if you are a Shopify user that does not operate on returns or adjustments then you will not have the related source tables. As such, you may use the below variable configurations to disable the respective downstream models. All variables are `true` by default. Only add the below configuration to your root `dbt_project.yml` if you are wishing to disable the models:
 
+```yml
+vars:
+  shopify__using_order_adjustment:  false  # true by default
+  shopify__using_order_line_refund: false  # true by default
+  shopify__using_refund:            false  # true by default
+```
+## (Optional) Step 6: Additional configurations
```
Comment (fivetran-joemarkiewicz) [inline README.md:76]:
```suggestion
## (Optional) Step 5: Additional configurations
```

File: README.md [hunk 5]
```diff
@@ -76,64 +90,61 @@ vars:
...
+    shopify_<default_source_table_name>_identifier: your_table_name 
+```
+</details>
 
-## Database support
-This package has been tested on BigQuery, Snowflake, Redshift, Postgres, and Databricks.
 
-### Databricks Dispatch Configuration
-dbt `v0.20.0` introduced a new project-level dispatch configuration that enables an "override" setting for all dispatched macros. If you are using a Databricks destination with this package you will need to add the below (or a variation of the below) dispatch configuration within your `dbt_project.yml`. This is required in order for the package to accurately search for macros within the `dbt-labs/spark_utils` then the `dbt-labs/dbt_utils` packages respectively.
+## (Optional) Step 7: Orchestrate your models with Fivetran Transformations for dbt Core™
```
Comment (fivetran-joemarkiewicz) [inline README.md:116]:
```suggestion
## (Optional) Step 6: Orchestrate your models with Fivetran Transformations for dbt Core™
```

File: packages.yml [hunk 6]
```diff
@@ -1,3 +1,7 @@
 packages:
-- package: fivetran/shopify_source
-  version: [">=0.6.0", "<0.7.0"]
+# - package: fivetran/shopify_source
+#   version: [">=0.7.0", "<0.8.0"]
+
+  - git: https://github.com/fivetran/dbt_shopify_source.git
+    revision: feature/package_updates
+    warn-unpinned: false
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:7]: Obligatory reminder to switch before merge

File: integration_tests/dbt_project.yml [hunk 7]
```diff
@@ -1,20 +1,22 @@
...
-    order_source:             "{{ ref('shopify_order_data') }}"
-    product_source:           "{{ ref('shopify_product_data') }}"
-    order_adjustment_source:  "{{ ref('shopify_order_adjustment_data') }}"
-    product_variant_source:   "{{ ref('shopify_product_variant_data') }}"
-    refund_source:            "{{ ref('shopify_refund_data') }}"
-    transaction_source:       "{{ ref('shopify_transaction_data') }}"
+    shopify_customer_source_identifier:          "shopify_customer_data"
+    shopify_order_line_refund_source_identifier: "shopify_order_line_refund_data"
+    shopify_order_line_source_identifier:        "shopify_order_line_data"
+    shopify_order_source_identifier:             "shopify_order_data"
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/dbt_project.yml:12]: Pretty sure this is the only change needed for circle to pass
```suggestion
    order_source:                         "{{ ref('shopify_order_data') }}"
```

**Review by fivetran-joemarkiewicz — APPROVED** [review]
@fivetran-reneeli thanks for these updates. Before this is good to go there are a few things we should update: 
- [x] CHANGELOG suggestion
- [ ] README edits
- [ ] CircleCi passes
- [ ] Regenerate docs

**Comment by fivetran-jamie** [comment]
@fivetran-reneeli should this be merged as well?

**Pull request merged**